### ISSUE TITLE
oauth redirection broken in web app

### ISSUE THREAD

**Issue by n-riesco** [plotly/falcon-sql-client#260]
After the user is authorised and popup window closes, the main window remains in login page.

![peek 2017-11-17 20-57](https://user-images.githubusercontent.com/6199391/32968707-1439f6e0-cbda-11e7-9abb-42eaaacb2fb3.gif)

**Comment by tarzzz** [comment]
@n-riesco Need some info regarding this:

  - What is the setting `SSL_ENABLED` here? Or `PLOTLY_CONNECTOR_SSL_ENABLED` .. 
  - Can you please send the cookies set on `localhost` .. 


For localhost, I would recommend starting docker container with `PLOTLY_CONNECTOR_SSL_ENABLED=false` and see if login works. 

In any case, this is not a blocker for on-prem, because login works on-prem on the latest build I tested today.

**Comment by scjody** [comment]
Agreed: as long as login works in On-Prem this isn't a release blocker for On-Prem.

@tarzzz If you become aware of any login issues that affect On-Prem, please log an issue in the 2.3.0 milestone of streambed (and also here if that's appropriate).
